insights/parsers/lscpu.py

```diff
@@ -0,0 +1,67 @@
+"""
+LsCPU - command ``lscpu``
+=========================
+
+This module provides the information about the CPU architecture using the output of the command ``lscpu``.
+"""
+from insights.parsers import SkipException, split_kv_pairs
+from insights.specs import Specs
+from .. import CommandParser, parser, LegacyItemAccess
```

Comment:
Better uniform the usage of `import` format in one single file, but not mixing the '..' (relative path) and full path.

Reply:
Done